lab/src/raft/append_entries.go

```diff
@@ -29,6 +30,7 @@ func (rf *Raft) AppendEntries(args *AppendEntriesArgs, reply *AppendEntriesReply
 	defer rf.mu.Unlock()
 
 	reply.Term = rf.currentTerm
+	rf.lastHeartBeat = time.Now()
```

Comment:
If this is not a valid heartbeat, I think lastHeartBeat should not be updated here